arangod/Cluster/ClusterMethods.cpp

```diff
@@ -3833,7 +3833,7 @@ arangodb::Result hotBackupCoordinator(ClusterFeature& feature, VPackSlice const
         break;
       }
       if (lockWait < 3600.0) {
-        lockWait *= 1.5;
+        lockWait *= 2.0;
       }
       std::this_thread::sleep_for(milliseconds(300));
     }
```

Comment:
I think this loop should also be aborted in case the current DB server instance is stopping.

Reply:
makes sense

Reply:
Added.